README.md

```diff
@@ -26,12 +26,23 @@ sudo apt-get install texlive-xetex texlive-fonts-extra fonts-yanone-kaffeesatz
 
 ### Exercises
 
-The codelabs only require ghc to be installed and in the path. Please read
-Codelab.hs and follow the instructions!
+The codelabs only require ghc to be installed and in the path as well as one of
+Stack or Cabal. Running `make` in each codelab directory will identify which of
+the tool is installed and use that to build the exercises as well as run the
+test.
```

Comment:
Not sure if this is really important, but I would probably clarify this by saying something like:

> The codelabs only require ghc to be installed and in the path as well as one of
> Stack or Cabal.

Step-by-step instructions will be provided to guide you through the installation process, at the point where these tools will be necessary.  In case you already have them installed 

> Running `make` in each codelab directory will identify which of
> the tool is installed and use that to build the exercises as well as run the
> test.

Reply:
Thanks for the feedback. Merged a PR with this wording.